modules/core_api/fsw/inc/cfe_version.h

```diff
@@ -28,21 +28,20 @@
 #define CFE_VERSION_H
 
 /* Development Build Macro Definitions */
-#define CFE_BUILD_NUMBER   80           /**< @brief Development: Number of development commits since baseline */
+#define CFE_BUILD_NUMBER   80 /**< @brief Development: Number of development git commits since CFE_BUILD_BASELINE */
 #define CFE_BUILD_BASELINE "v7.0.0-rc4" /**< @brief Development: Reference git tag for build number */
 
-/* Version Macro Definitions updated for official releases only */
-#define CFE_MAJOR_VERSION 6 /**< @brief Major release version (Former for Revision == 99) */
-#define CFE_MINOR_VERSION 7 /**< @brief Minor release version (Former for Revision == 99) */
-#define CFE_REVISION \
-    99 /*!< @brief * Set to 0 on OFFICIAL releases, and set to 99 on development versions. Revision number.      */
+/* See \ref cfsversions for definitions */
+#define CFE_MAJOR_VERSION 6
+#define CFE_MINOR_VERSION 7
+#define CFE_REVISION      99
 
 /*!
  * @brief Mission revision.
  *
- * Reserved for mission use to denote patches/customizations as needed.
+ * Reserved for mission use to denote patches/customizations as needed. 
  */
-#define CFE_MISSION_REV 0
+#define CFE_MISSION_REV 0xFF
 
```

Comment:
Are we good with this or should we add something like 0-244 being "reserved" for missions but 0xFF *may* indicate development

Reply:
1-254 you mean?  0 and 255 we've claimed.  You could just reference the issue?  Not sure how helpful that is though.